.github/workflows/ci.yml

```diff
@@ -323,7 +323,7 @@ jobs:
           go-version: 1.22.7
 
       - name: Fetch deps
-        run: sudo apt-get -q update && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential squashfs-tools squashfuse fuse-overlayfs fakeroot fuse2fs libseccomp-dev cryptsetup dbus-user-session
+        run: sudo apt-get -q update && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential fakeroot libseccomp-dev cryptsetup dbus-user-session
```

Comment:
There are more places in .github/workflows/ci.yml where these packages should be deleted.

Reply:
Okay, will do the cleanup for that separately I think.

Reply:
If you'd like to leave it separate the easiest way is to put it an another commit in this PR.